salt/modules/virt.py

```diff
@@ -1878,25 +1879,21 @@ def update(
     :param password: password to connect with, overriding defaults
 
     :param boot:
-        Specifies kernel for the virtual machine, as well as boot parameters
-        for the virtual machine. This is an optionl parameter, and all of the
-        keys are optional within the dictionary. If a remote path is provided
-        to kernel or initrd, salt will handle the downloading of the specified
-        remote fild, and will modify the XML accordingly.Boot vm with UEFI,
-        loader and nvram can be specified to achieve the goal. To remove any
-        boot parameters, pass an None object to the relevant parameter, for instance: "kernel": None
+        Boot parameter definition is documented in the :func:`init` function. To update any
+        boot parameters, specify the new path for each. To delete any boot parameters,
+        pass a None object, for instance: 'kernel': None.
 
         .. code-block:: python
 
             {
-                'kernel': '/root/f8-i386-vmlinuz',
-                'initrd': '/root/f8-i386-initrd',
-                'cmdline': 'console=ttyS0 ks=http://example.com/f8-i386/os/'
-                'loader': /usr/share/OVMF/OVMF_CODE.fd
-                'nvram': /usr/share/OVMF/OVMF_VARS.ms.fd
+                'kernel': '/root/new-vmlinuz',
+                'initrd': '/root/new-initrd',
+                'cmdline': 'console=ttyS0 ks=http://example.com/f8-i386/os/',
+                'loader': '/usr/share/new/OVMF_CODE.fd',
+                'nvram': '/usr/share/new/OVMF_VARS.ms.fd'
             }
 
-        .. versionadded:: 3000
+        .. versionadded:: sodium
```

Comment:
You could then replace the whole parameter description with something like this:

```suggestion
        Specifies kernel for the virtual machine, as well as boot parameters
        for the virtual machine. This is an optional parameter, and all of the
        keys are optional within the dictionary.
        
        Refer to :ref:`init-boot-def` for the complete value description.
        
        To remove any boot parameters, pass an None object to the relevant parameter, for instance: "kernel": None
        To update any boot parameters, specify the new path for each. To delete any boot parameters,
        pass a None object, for instance: ``'kernel': None``.
        
        .. versionadded:: 3000
```